### PR TITLE
Badge and button components

### PR DESCRIPTION
**Pull request by Manju-sree**
Added badge component of 2 variants.
1. Badge on Icon
2. Badge on Avatar

Added button components with 4 different variants.
1. Solid button
2. link with outline button
3. button with icons
4. float button

### CODE REVIEW

File: Components/Badge/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const button2 = document.querySelector(".btn-hide-bdg ");
```
Comment (ultronjs) [inline Components/Badge/index.js]: variable names are quite confusing

File: Components/Badge/badges.css [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+}
+
+.badge-btn {
+    position: relative;
+    font-size: 0.8rem;
+    font-weight: bolder;
+    border: none;
+    border-radius: 0.2rem;
+}
+
```
Comment (ultronjs) [inline Components/Badge/badges.css:23]: be consistent with class naming:- .Badge-count and .badge-btn. One is starting with capital and other with small

**Review by ultronjs — APPROVED** [review]

**Review by ajit-sharmaGH — APPROVED** [review]
Hello I went through your code, it looks fine a suggestion is to use code formatter before making a pull request!

File: Components/Badge/index.js (hunk 1 above)
Comment (Manju-sree) [reply Components/Badge/index.js]: Variable names are changed.. plz check

File: Components/Badge/badges.css (hunk 2 above)
Comment (Manju-sree) [reply Components/Badge/badges.css:23]: yeah everything i have made as small now.. plz check

File: Components/Badge/index.js (hunk 1 above)
Comment (ultronjs) [reply Components/Badge/index.js]: nice

File: Components/Badge/badges.css (hunk 2 above)
Comment (ultronjs) [reply Components/Badge/badges.css:23]: good

**Comment by ultronjs** [comment]
Good to go ahead.

**Comment by Manju-sree** [comment]
Thank You!

**Pull request merged**